modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-pgt-image-registry_{context}"]
+= Configuring the image registry using PolicyGenTemplate CRs
+
+You can use `PolicyGenTemplate` to apply to create the PV and PVC and patch `imageregistry` configuration. Select the appropriate `PolicyGenTemplate` for each `source-cr`. See Additional Resources for more help.
+
+.Prerequisites
+
+* You have installed and configured Zero Touch Provisioning (ZTP). For information about this, see the topic on ZTP in Additional resources.
+
+.Procedure
+
+. Add the following YAML to the `StorageClass.yaml` file to configure the image registry:
```

Comment:
This is incorrect. The `StorageClass.yaml` is the name of the CR file that the you add to the PGT.
 
```suggestion
. Configure the storage class, persistent volume claim, persistent volume, and image registry configuration in the appropriate `PolicyGenTemplate` CR. For example, to configure an individual site, use the following YAML:
```

Reply:
Done.